.github/workflows/backend-formatting.yml

```
@@ -0,0 +1,55 @@
+name: rust-formatting
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
```

```suggestion
name: rust

on:
  [pull_request]
```
It does not only formatting but also clippy(linting) and testing.
So I think just naming it "rust" is more appropriate.

And... I suppose running on pull_request is enough, isn't it?

---

I see. i'll change.

---

PR立てる前にCI落ち認知できた方が嬉しくない？

---

確かにそうかも。

---

私は完璧で究極のDraft Pullrequest過激派なので